src/popup/searchModule.js

```diff
@@ -1,13 +1,20 @@
 import { elements } from './base';
-import { unicodeToString, providerLogos, addLoadMoreButton } from './helper';
+import {
+  unicodeToString, providerLogos, addLoadMoreButton,
+} from './helper';
 import { activatePopup } from './infoPopupModule';
 import { removeSpinner } from './spinner';
 // eslint-disable-next-line import/no-cycle
 import bookmarkImage from './bookmarkModule';
-import { showNotification, removeChildNodes, restoreInitialContent } from '../utils';
+import {
+  showNotification, removeChildNodes, restoreInitialContent,
+} from '../utils';
 
 const Masonry = require('masonry-layout');
 
+let removed = false;
```

Comment:
There is no need to use a variable. `node.classList.remove()` and `node.classList.add()` does not throw any exception if you are removing a class not present or if you are adding an already present class respectively.